heron/tools/ui/resources/static/js/plan-controller.js

```diff
@@ -184,8 +184,8 @@ function PlanController(baseUrl, cluster, environ, toponame, physicalPlan, logic
     highlightFocusedElements();
   };
 
-  // treat any click that bubbles up unhandled as intent to clear the focus
-  $(document).on('click', ".plans .graphics", function () {
+  // treat double click as intent to clear the focus
```

Comment:
besides comments, shall we add the instruction somewhere, so that users can easily find it and know how to operate the UI webpage?

Reply:
We dont have an instruction page for heorn UI I think currently. Could be useful to have one though.